### PR TITLE
add name, http_proxy, https_proxy and no_proxy to docker info

### LINKED ISSUE #1029: HttpProxy not showing up with GET /info
My docker host is  behind a proxy. I have set up http_proxy as mentioned in the docs - https://docs.docker.com/articles/systemd/.
But this doesnt show up when I do a GET /info using the remote API. 

"HttpProxy": "",
    "HttpsProxy": "",
    "NoProxy": ""

I am facing a problem similar to the one mentioned in 
https://forums.docker.com/t/proxy-issue-in-swarm/1221

I am not able to start a container, see logs of a container because it keeps throwing a proxy error.

### PR DESCRIPTION
**Pull request by vieux**
Close #1029

### CODE REVIEW

**Comment by vieux** [comment]
ping @aluzzardi @abronan @vinaychatra

File: api/handlers.go [hunk 1]
```diff
@@ -39,6 +40,19 @@ func getInfo(c *context, w http.ResponseWriter, r *http.Request) {
 		MemTotal:          c.cluster.TotalMemory(),
 	}
 
+	if httpProxy := os.Getenv("http_proxy"); httpProxy != "" {
```
Comment (aluzzardi) [inline api/handlers.go]: Is the `if` really necessary? Wouldn't `HttpProxy: os.Getenv("http_proxy")` above have the same resullt?

Comment (vieux) [inline api/handlers.go]: probably not, to be sincere I just copy pasted from docker/docker...

I'll update

**Comment by aluzzardi** [comment]
LGTM

**Pull request merged**